mobile/src/main/java/org/openhab/habdroid/model/Item.kt

```diff
@@ -159,3 +158,10 @@ fun String?.toItemType(): Item.Type {
         Item.Type.None
     }
 }
+
+fun Item.canBeToggled(): Boolean {
```

Comment:
This does not make much sense as extension function, as it can be added to the actual class.